samples/BoltOn.Samples.WebApi/Startup.cs

```diff
@@ -22,23 +23,27 @@ public Startup(IConfiguration configuration)
 
         public void ConfigureServices(IServiceCollection services)
         {
-            services.AddMvc().SetCompatibilityVersion(CompatibilityVersion.Version_2_2);
-			services.BoltOn(options =>
-			{
-				options.BoltOnEFModule();
-				options.BoltOnAssemblies(typeof(PingHandler).Assembly, typeof(StudentRepository).Assembly);
-			});
-
-			services.AddDbContext<SchoolDbContext>(options =>
-			 {
-				 options.UseSqlServer("Data Source=127.0.0.1;initial catalog=Testing;persist security info=True;User ID=sa;Password=$Password1;");
-			 });
-		}
+            services.AddMvc().SetCompatibilityVersion(CompatibilityVersion.Version_2_2);
+            services.BoltOn(options =>
+            {
+                options.BoltOnEFModule();
+                options.BoltOnCosmosDbModule();
+                options.BoltOnAssemblies(typeof(PingHandler).Assembly, typeof(StudentRepository).Assembly);
+            });
+
+            services.AddDbContext<SchoolDbContext>(options =>
+             {
+                 options.UseSqlServer("Data Source=127.0.0.1;initial catalog=Testing;persist security info=True;User ID=sa;Password=$Password1;");
+             });
+
+            services.Configure<CosmosDbSettings>(options => Configuration.GetSection("CosmosDbSettings").Bind(options));
```

Comment:
try to use options pattern for CosmosDbContext

Reply:
@gokulm : i forgot on what we discussed abt the change on the cosmosrepo , will catch up with u on monday.

Reply:
@skankanvar add an extension method AddCosmosDbContext with CosmosDbOptions as a parameter. Retrieve the properties from options and set it in the dbcontext and make the dbcontext singleton.